src/targets/neutronrc_900.ini

```diff
@@ -0,0 +1,33 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+
+# ********************************
+# Receiver targets
+# ********************************
+
+[env:NeutronRC_900_RX_via_UART]
+extends = env_common_esp82xx
+upload_speed = 460800
+monitor_speed = 460800
+build_flags =
+	${env_common_esp82xx.build_flags}
+	${common_env_data.build_flags_rx}
+	-D TARGET_EXPRESSLRS_PCB_RX_V3
+	-D TARGET_100mW_MODULE
+src_filter = ${env_common_esp82xx.src_filter} -<tx_*.cpp>
+
+[env:NeutronRC_900_RX_via_BetaflightPassthrough]
+extends = env:NeutronRC_900_RX_via_UART
+upload_speed = 460800
```

Comment:
these have gotta be 420000

Reply:
Looks like some of the diy targets will also need updating.

Reply:
Why??